apps/jira/jira-app/src/constants.ts

```diff
@@ -1,13 +1,15 @@
 /* global process */
 
 let LAMBDA_URI = 'https://api.jira.ctfapps.net';
+let CLIENT_ID = 'XD9k9QU9VT4Rt26u6lbO3NM0fOqvvXan';
 
 if (process.env.nodeEnv === 'development') {
```

Comment:
On a second thought - might be better to have this as `process.env.nodeEnv !== 'development'`

Reply:
I have a feeling changing that would require some other work. It's safer to assume it's always production for now and explicitly state when we are in development.